crates/edr_napi/test/optimism.ts

```diff
@@ -0,0 +1,159 @@
+import chai, { assert } from "chai";
+import chaiAsPromised from "chai-as-promised";
+
+import {
+  ContractAndFunctionName,
+  EdrContext,
+  L1_CHAIN_TYPE,
+  l1ProviderFactory,
+  MineOrdering,
+  SubscriptionEvent,
+  // HACK: There is no way to exclude tsc type checking for a file from the
```

Comment:
There is no way to exclude files from `tsc` via the command-line (https://github.com/microsoft/TypeScript/issues/46005)